framework/src/wifi_manager/Kconfig

```diff
@@ -101,5 +101,11 @@ config DISABLE_EXTERNAL_AUTOCONNECT
 	bool "Disable external autoconnect"
 	default n
 
+config ENABLE_DRIVER_IOCTL
```

Comment:
I don't think this config is necessary.
1. if user doesn't throw ioctl with specific args, ioctl will not be called
2. File ops is not enough to control h/w, so framework should control driver through ioctl. Is it really possible to control h/w without any of ioctl???? open read write close are enough?
3. If you want restrict some kind of get status or set power only, you should rename this config to specific one. Current is too generic.

Reply:
Thanks for your opinion, Here are answers to your requests. I think answer for 3 can cover your question.
Yes now  operations only get status and  set power which is not mandatory, require ioctl interface. 
So we can make ioctl optional. 
HAL interface of wi-fi has defined basic operations like init, deinit, start sta commands. 
However as requirements from product are added, there are needs to add additional features to wi-fi driver. 
It's difficult to add wi-fi HAL interface every time when requirements are added. So I made ioctl  to apply requirement rapidly.